cssom-view-1/Overview.bs

```diff
@@ -2015,6 +2015,18 @@ Issue: In what order are scrollend events dispatched? Ordered based on scroll st
 </table>
 
 
+Post-Layout State Snapshotting {#post-layout-snapshot}
+======================================================
+
+Some CSS features use post-layout state, like scroll position, as input to the next style and layout update.
+
+When asked to <dfn export>run snapshot post-layout state steps</dfn> for a {{Document}} <var>doc</var>, run these steps:
+
+1. For each CSS feature that needs to snapshot post-layout state, take a snapshot of the relevant state.
```

Comment:
This probably needs to reference `doc` somehow? Maybe "take a snapshot of the relevant state in `doc`" or so?

Reply:
Done